assets/js/blocks/cart-checkout/checkout/block.js

```diff
@@ -92,251 +95,255 @@ const Block = ( {
 	} = useShippingRates();
 
 	return (
-		<CheckoutProvider isEditor={ isEditor }>
-			<SidebarLayout className="wc-block-checkout">
-				<Main>
-					<ExpressCheckoutFormControl />
-					<CheckoutForm>
-						<FormStep
-							id="contact-fields"
-							className="wc-block-checkout__contact-fields"
-							title={ __(
-								'Contact information',
-								'woo-gutenberg-products-block'
-							) }
-							description={ __(
-								"We'll use this email to send you details and updates about your order.",
-								'woo-gutenberg-products-block'
-							) }
-							stepHeadingContent={ () => (
-								<Fragment>
-									{ __(
-										'Already have an account? ',
-										'woo-gutenberg-products-block'
-									) }
-									<a href="/wp-login.php">
-										{ __(
-											'Log in.',
-											'woo-gutenberg-products-block'
-										) }
-									</a>
-								</Fragment>
-							) }
-						>
-							<TextInput
-								type="email"
-								label={ __(
-									'Email address',
-									'woo-gutenberg-products-block'
-								) }
-								value={ contactFields.email }
-								autoComplete="email"
-								onChange={ ( newValue ) =>
-									setContactFields( {
-										...contactFields,
-										email: newValue,
-									} )
-								}
-								required={ true }
-							/>
-							<CheckboxControl
-								className="wc-block-checkout__keep-updated"
-								label={ __(
-									'Keep me up to date on news and exclusive offers',
-									'woo-gutenberg-products-block'
-								) }
-								checked={ contactFields.keepUpdated }
-								onChange={ () =>
-									setContactFields( {
-										...contactFields,
-										keepUpdated: ! contactFields.keepUpdated,
-									} )
-								}
-							/>
-						</FormStep>
-						{ SHIPPING_ENABLED && (
+		<ValidationContextProvider>
+			<CheckoutProvider isEditor={ isEditor }>
+				<SidebarLayout className="wc-block-checkout">
```

Comment:
`ValidationContextProvider` is wrapping `CheckoutProvider` so that the validation context is available to the checkout processing event.